### DISCUSSION TITLE
Getting Started

### DISCUSSION THREAD

**Discussion by codetyri0n** [JuliaImages/Images.jl#1045] [General]
Greetings to the respected maintainers ,
My name is Sriram and I am a computer science sophomore from India and would like to get started with contributing to Julia Images . I am right now getting familiarized with Julia and am passionate about machine learning and I have wanted to contribute to the ecosystem ever since I got to know about the language and community. Coincidentally a few of the research faculty at my institution use Julia.
My prior experience with regards to image processing lie in a workshop as part of a symposium I took part last month as a speaker.I demonstrated how a small scale image processing model works with training and testing using Tensorflow.(https://github.com/codetyri0n/ML-Workshop)

I hope I can contribute something worthwhile and request your guidance for me to do the same.Looking forward to your response.
Thank you.

**Comment by mkitti** [comment]
Welcome. How can we help?

**Reply by codetyri0n** [reply]
Hi , thanks for your response!
I just wanted to drop an introduction in this space as I am just getting started , I am following the instructions and guidelines set in the CONTRIBUTING.md document. I think this discussion page would help me if I have any queries about the same.
Regards
